### PR TITLE
Create logout popup

### PR DESCRIPTION
**Pull request by ko-yaroslav**
This PR based on #8

### CODE REVIEW

File: src/containers/Auth/story.js [hunk 1]
```diff
@@ -46,3 +46,9 @@ export const resetPasswordStory = async ({ data, trans }, dispatch) => {
     dispatch(successNotification(trans('notification.success.auth.password.reset')));
   }
 };
+
+export const logoutStory = async (dispatch) => {
+  await dispatch(logoutAction());
+
+  history.push(ROUTE_TO_LOGIN);
```
Comment (yaroslavfedyshyn) [inline src/containers/Auth/story.js]: @ko-yaroslav redirect to home page

File: src/containers/Auth/ConfirmLogoutDialog/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Popup from '../../../components/Popup';
+import { logoutStory } from '../story';
+
+const ConfirmLogoutDialog = (props) => (
+  <Popup
+    trigger={<a href="#" id="logoutBtn">Logout</a>}
```
Comment (yaroslavfedyshyn) [inline src/containers/Auth/ConfirmLogoutDialog/index.js]: @ko-yaroslav we don't need here trigger button. we will show popup in other way

File: src/components/Popup/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+export default ({
+  trigger,
+  title,
+  content,
+  leftBtn,
+  rightBtn,
+  onClickHandler,
+}) => (
+  <Popup trigger={trigger} modal closeOnDocumentClick>
```
Comment (yaroslavfedyshyn) [inline src/components/Popup/index.js]: @ko-yaroslav pass open property. remove trigger prop

File: src/containers/Auth/ConfirmLogoutDialog/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    title={'Log out'}
+    content={'Are you sure you want to logout?'}
+    leftBtn={'Cancel'}
+    rightBtn={'Log out'}
+    onClickHandler={props.onLogoutHandler}
```
Comment (yaroslavfedyshyn) [inline src/containers/Auth/ConfirmLogoutDialog/index.js]: @ko-yaroslav rename to onSubmit

File: src/containers/Header/Menu/components/Menu.js [hunk 5]
```diff
@@ -60,7 +60,8 @@ const Menu = (props) => {
                       <div className="mr-50" id={'profile-logout-btn'}>
                         <Link to={ROUTE_TO_PROFILE} id="profileBtn">Profile</Link>
                         <span style={{ color: 'white' }}> / </span>
-                        <Link to={ROUTE_TO_LOGOUT} id="logoutBtn"> Logout</Link>
```
Comment (yaroslavfedyshyn) [inline src/containers/Header/Menu/components/Menu.js]: @ko-yaroslav add onClick handler which will set openLogoutConfirmPopup: true to local state

File: src/containers/Header/Menu/components/Menu.js [hunk 6]
```diff
@@ -60,7 +60,8 @@ const Menu = (props) => {
...
+
+                        <ConfirmLogoutDialog />
```
Comment (yaroslavfedyshyn) [inline src/containers/Header/Menu/components/Menu.js]: @ko-yaroslav show popup with condition. if openLogoutConfirmPopup show popup - else hide

**Comment by yaroslavfedyshyn** [comment]
@ko-yaroslav after changes fix conflicts

File: src/containers/Auth/story.js (hunk 1 above)
Comment (ko-yaroslav) [reply src/containers/Auth/story.js]: Done.

File: src/containers/Auth/ConfirmLogoutDialog/index.js (hunk 2 above)
Comment (ko-yaroslav) [reply src/containers/Auth/ConfirmLogoutDialog/index.js]: Done.

File: src/components/Popup/index.js (hunk 3 above)
Comment (ko-yaroslav) [reply src/components/Popup/index.js]: Done.

File: src/containers/Auth/ConfirmLogoutDialog/index.js (hunk 4 above)
Comment (ko-yaroslav) [reply src/containers/Auth/ConfirmLogoutDialog/index.js]: Done.

File: src/containers/Header/Menu/components/Menu.js (hunk 5 above)
Comment (ko-yaroslav) [reply src/containers/Header/Menu/components/Menu.js]: Done.

File: src/containers/Header/Menu/components/Menu.js (hunk 6 above)
Comment (ko-yaroslav) [reply src/containers/Header/Menu/components/Menu.js]: Done.

**Comment by ko-yaroslav** [comment]
> @ko-yaroslav after changes fix conflicts

Done.

**Pull request merged**